### DISCUSSION TITLE
dataset class loader

### DISCUSSION THREAD

**Discussion by EvelyneCalista** [TorchIO-project/torchio#1157] [Q&A]
hi, is that possible to let the **_SubjectsDataset_** return original image and its augmentation transformation altogether, without save the augmented data locally first?

**Comment by romainVala** [comment]
Hello
not directly, but it is easy to code with one extra line
```
suj_transform = torchio_transform(suj_orig)
suj_orig.add_image(suj_transform.ImageName, 'TransformName')

```
